Build not outputting exe

Not sure what's going, system is windows
cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release
-- Trying to find Qt 6...
-- Could NOT find WrapVulkanHeaders (missing: Vulkan_INCLUDE_DIR)
-- Success! Configuration details:
-- App name: Notes
-- App version: 2.1.0
-- Qt version: 6.5.0
-- Update checker: ON
-- Build type: Release
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Programming/Git/notes/build

--install build --prefix build
-- Install configuration: "Release"
CMake Error at build/cmake_install.cmake:36 (file):
file INSTALL cannot find "C:/Programming/Git/notes/build/Notes.exe": No error.

Has same issue building to different directory as well, any ideas?

--- a/docs/build_on_linux.md
+++ b/docs/build_on_linux.md
@@ -40,7 +40,7 @@ cd build
 
 After that, we're ready to build Notes!
 
-Invoke CMake to build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
+Invoke CMake to configure and build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
 
 ```shell
 cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release

--- a/docs/build_on_macos.md
+++ b/docs/build_on_macos.md
@@ -34,10 +34,11 @@ export CMAKE_BUILD_PARALLEL_LEVEL=$(sysctl -n hw.logicalcpu)
 
 After that, we're ready to build Notes!
 
-Invoke CMake to build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
+Invoke CMake to configure and build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
 
 ```shell
-cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release'
+cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release
+cmake --build build
 ```
 
 Alternatively, if you want to target both `x86_64` and `arm64` (Apple Silicon) architectures in a single, universal binary, invoke CMake like this instead *(please note that this might be unsupported on Qt versions older than 6)*:

--- a/docs/build_on_windows.md
+++ b/docs/build_on_windows.md
@@ -7,7 +7,7 @@ Windows 7 (or newer) is required to build Notes.
 Additionally, you need to install the following tools/components:
 
 - [Git](https://gitforwindows.org/)
-- [Microsoft Visual C++](https://visualstudio.microsoft.com/downloads) `>= 2015` *(only the build tools are required)*
+- [Microsoft Visual C++](https://visualstudio.microsoft.com/downloads) `>= 2017` *(only the build tools are required)*
 - [Qt](https://www.qt.io/download-qt-installer) `>= 5.9` *(using the latest version is recommended)*
   - [CMake](https://cmake.org/download/) *(can be installed via the Qt installer)*
   - [Ninja](https://ninja-build.org/) *(can be installed via the Qt installer)*
@@ -36,7 +36,7 @@ Depending on what version of these softwares you have installed, you may need to
 Let's begin by invoking the `vcvars64.bat` script from MSVC 2017, which will set some helpful environment variables for us:
 
 ```shell
-@call "C:\Program Files (x86)\Microsoft Visual Studio\2015\BuildTools\VC\Auxiliary\Build\vcvars64.bat"
+@call "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat"
 ```
 
 Now, we need to add CMake, Ninja, Qt sources and tools to our `Path` environment variable:

--- a/docs/build_on_windows.md
+++ b/docs/build_on_windows.md
@@ -56,10 +56,17 @@ set CMAKE_BUILD_PARALLEL_LEVEL=%NUMBER_OF_PROCESSORS%
 
 After that, we're ready to build Notes!
 
-Invoke CMake to build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
+Invoke CMake to configure and build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
 
 ```shell
 cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release
+cmake --build build
+```
+
+<!-- FIXME: Avoid hardcoding the output folder to `bin` on Windows? -->
+Now, let's move the `Notes.exe` executable we just built into a separate folder called `bin` inside the `build` folder, where later we will copy other libraries required to run the app:
+
+```shell
 cmake --install build --prefix build
 ```
 

